jest.setup.js

```diff
@@ -0,0 +1,17 @@
+/*
```

Comment:
Can we rename this file to `test-setup.js` to be consistent?

Reply:
Updated in 2d01ba7